model/validation.go

```diff
@@ -228,8 +579,33 @@ func ValidateDestinationPolicy(msg proto.Message) error {
 	if !ok {
 		return fmt.Errorf("Cannot cast to destination policy")
 	}
+
+	var retVal error
```

Comment:
nit: `retVal` is non-descriptive. I think the convention for `multierror` is generally `errs`.